lib/_sendEvent.ts

```diff
@@ -143,9 +146,21 @@ function bulkSendEvent(
   apiKey: string,
   userAgent: string,
   endpointOrigin: string,
-  events: InsightsEvent[]
+  events: InsightsEvent[],
+  additionalHeaders: InsightsAdditionalEventParams["headers"] = {}
```

Comment:
technically query params, but I guess it's equivalent

Reply:
Yes I wondered but I thought that although sent as query parameters, they are first custom headers.